core/src/main/java/com/airbnb/aerosolve/core/transforms/NormalizeFloatFeatureTransform.java

```diff
@@ -0,0 +1,44 @@
+package com.airbnb.aerosolve.core.transforms;
+
+import com.airbnb.aerosolve.core.FeatureVector;
+import com.airbnb.aerosolve.core.util.Util;
+
+import com.typesafe.config.Config;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+// L2 normalizes a float feature
+public class NormalizeFloatFeatureTransform extends Transform {
+  private String fieldName1;
+
+  @Override
+  public void configure(Config config, String key) {
+    fieldName1 = config.getString(key + ".field1");
```

Comment:
how about adding `output` so that if user can have the option to output to a different feature family and keep both normalized and unnormalized features